crates/wallet/src/wallet/signer.rs

```diff
@@ -481,7 +481,7 @@ impl InputSigner for SignerWrapper<PrivateKey> {
                     && sign_options.sign_with_tap_internal_key
                     && x_only_pubkey == psbt_internal_key
                 {
-                    let (hash, hash_ty) = Tap::sighash(psbt, input_index, None)?;
+                    let (hash, hash_ty) = compute_tap_sighash(psbt, input_index, None)?;
```

Comment:
nit: I would suggest keeping the `rust-bitcoin` standard of `sighash` and `sighash_type`
_same applies to other lines below, but won't spam it_